frontend/src/main/components/Jobs/PagedJobsTable.js

```diff
@@ -3,7 +3,7 @@ import OurTable, { PlaintextColumn, DateColumn } from "main/components/OurTable"
 import { Button } from "react-bootstrap";
 import { useBackend } from "main/utils/useBackend";
 
-export default function PagedJobsTable() {
+const PagedJobsTable = ({jobs: jobsProp}) => {
```

Comment:
Renaming a prop to append “Props” doesn’t tell you anything. If it’s for stories, give it a meaningful name that says so.